hera_sim/simulate.py

```diff
@@ -367,118 +537,292 @@ def chunk_sim_and_save(
     # TODO: write a deprecated wrapper function
     def add_eor(self, model, **kwargs):
         """
-        Add an EoR-like model to the visibilities. See :meth:`add` for
+        Add an EoR-like model to the visibilities. See .. meth:: `add` for
```

Comment:
I'm pretty sure the original is correct here? The `.. meth::` form is for its own line.

Reply:
That's right. I've reverted this change.